.github/workflows/unittests.yaml

```
@@ -29,27 +31,18 @@ jobs:
           lfs: true
       - name: Checkout LFS objects
         run: git lfs checkout        
-      - name: Setup Scala
-        uses: olafurpg/setup-scala@v12
-        with:
-          java-version: adopt-openj9@1.8.0-292
-      - name: Cache .ivy2
-        uses: actions/cache@v4
-        with:
-          path: ~/.ivy2
-          key: ${{ runner.os }}-ivy2-${{ hashFiles('**/build.sbt') }}
-      - name: Cache .sbt
-        uses: actions/cache@v4
+      - name: Setup cache
+        uses: coursier/cache-action@v6
+      - name: setup Scala
+        uses: coursier/setup-action@v1
         with:
-          path: ~/.sbt
-          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.properties') }}
+          apps: scala:2.13.14 scalac:2.13.14 scaladoc:2.13.14 sbt:1.10.1
+          jvm: ${{ matrix.jvm }}
       - name: Unit Tests
         run: |
           set -e
           sbt +clean coverage +test
```

chore: while you're here... we don't build 2.12 anymore, so any "+<command>" can be updated
```suggestion
          sbt clean coverage test
```